pkg/descheduler/strategies/node_affinity_test.go

```diff
@@ -184,6 +185,7 @@ func TestRemovePodsViolatingNodeAffinity(t *testing.T) {
 			policyv1.SchemeGroupVersion.String(),
 			false,
 			tc.maxPodsToEvictPerNode,
+			tc.maxNoOfPodsToEvictPerNamespace,
```

Comment:
Given `maxNoOfPodsToEvictPerNamespace` is not set in any test case, you can just pass `nil`

Reply:
Yet. you are right. it should pass `nil` :)

Reply:
Fix done ~~ Please check :)